ui.tests/test-module/specs/tabsontop/tabsontop.runtime.spec.js

```diff
@@ -263,13 +263,13 @@ describe("setFocus on the first invalid field on submission", () => {
     });
 
     it("check if invalid field gets focus on submit button click", () => {
-           cy.get('button').should('be.visible').click().then(() => {
+           cy.get('button').last().should('be.visible').click().then(() => {
```

Comment:
Please don't change anything in an existing test case, unless there is backward incompatible change

Reply:
Previously we did not include the questionMark div if long description was not set, but since the setProperty can be called at runtime and even if long description was not set we need to set it in runtime, then we need to have this div. As per this change https://github.com/adobe/aem-core-forms-components/pull/1274/files#:~:text=%3Cbutton%20class%3D%22%24%7BbemBlock%7D__questionmark%22%20style%3D%22%24%7BlongDescription%20%3F%20%27%27%20%3A%20%27display%3Anone%3B%27%7D%20title%3D%22Help%20text%22%3E

Now in the test cases the selectors were wrong as it was assuming that there would only be one button in the form, so I had to update the selectors. 
Not sure if this would be backward incompatible though

Reply:
But this would pollute the HTML with unncessary markup for each field, thus can impact HTML parsing even for use-cases where description is not set at runtime, which is incorrect.

Ideally, at runtime, we should generate the markup if description is set at runtime.